packages/cozy-pouch-link/src/replicateOnce.js

```diff
@@ -73,11 +62,6 @@ export const replicateOnce = async pouchManager => {
         pouchManager.storage,
         pouchManager.client
       )
-      if (seq) {
-        // We only need the sequence for the second replication, as PouchDB
-        // will use a local checkpoint for the next runs.
-        await pouchManager.storage.destroyDoctypeLastSequence(doctype)
-      }
```

Comment:
This changes the behaviour: before we were using the lastSeq only for 2nd replication (because the first one was not done through pouch replication). Now, we rely on it for all the replications. I guess it's ok, but I'm wondering if any side effect could occur

Reply:
I don't understand. With this we still do the first replication without pouch replication. It only depends on `initialReplication` parameter which does not change

Reply:
Ok, I see. Let's not change this behavior. I put the sequence removal back

Reply:
Well if you put the `lastSeq` removal as before, you will break the shared drive next "replications", since they won't have any `lastSeq` to rely on, as they don't use the actual replication protocol